esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -66,6 +66,9 @@
 //! }
 //! ```
 
+#![allow(missing_docs)] // TODO: Remove when able
```

Comment:
I don't think we should introduce this again

Reply:
Sorry, I think this was an accidental add when I merged main into my branch. Will remove